content/articles/2017-11-21-usethis-1-0-0.Rmd

```diff
@@ -0,0 +1,144 @@
+---
+title: usethis 1.0.0 (and 1.1.0)
+author: Hadley Wickham
+date: '2017-11-21'
+slug: usethis-1.0.0
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/IClZBVw5W5A
+  author: Todd Quackenbush
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're pleased to announce the release of [usethis 1.0.0](https://github.com/r-lib/usethis/releases/tag/v1.0.0) 
+on CRAN (now [1.1.0](https://github.com/r-lib/usethis/releases/tag/v1.1.0), following some small improvements discovered after release). usethis is designed to smooth package development workflows by automating everything that can be automated. Many of the functions were previously part of devtools, but have been extracted out in their own package to give them more room to grow.
```

Comment:
I'm kind of obsessive about hyphenating [stacked modifiers](http://www.quickanddirtytips.com/education/grammar/stacked-modifiers) (e.g. `package-development` modifying `workflow`), but that's possibly stylistic.